### ISSUE TITLE
Python modules do not build against Boost.Python>1.65.1

### ISSUE THREAD

**Issue by rmaronson** [rdiankov/openrave#551]
Building with Boost.Python version 1.65.1 or greater is broken, as the `boost::python::numeric` namespace has been removed in favor of `boost::python::numpy`. See [version notes](http://www.boost.org/users/history/version_1_65_1.html).

**Comment by marynelv** [comment]
Yup. I can confirm this. I was able to build against boost 1.58 (1.57 has some other issue as well).

**Comment by morxa** [comment]
This also fixes #542.

**Comment by lianghongzhuo** [comment]
Any plan to Migrate from boost::numeric to boost:numpy?

**Comment by cielavenir** [comment]
we are migrating to pybind11. no document how to install it yet, but you could learn from https://github.com/cielavenir/mujin_recruiting/blob/pybind11/cookbooks/mujin/recipes/default.rb#L106

**Comment by lianghongzhuo** [comment]
@cielavenir Can you successfully compile the master branch in your computer? If so,  what's your operating system and version? Thanks.

**Comment by cielavenir** [comment]
@lianghongzhuo Debian stretch/buster, Ubuntu bionic/focal
